README.md

```diff
@@ -1 +1,9 @@
 # cs-5666-scatt-gp
+
+## Demo Instructions
+* Login to the Appalachian student server.
```

Comment:
@KaraBeason Yes, but I think with its current format, you have to have a JRE (the 'make scatt' compiles the application files) and make installed. I tried it on a lab PC and it didn't work, so, in the interest of keeping it simple, I thought this might be best for now. I added an issue for us to make a .jar file that will run independent of the OS to work on the next iteration - something the user could download and double click on to open. But let me know if I'm wrong about this - we could create a branch that has the files compiled already and change the instructions to say make is a requirement?